sdk/python/kfp/cli/cli.py

```diff
@@ -20,28 +20,26 @@
 from .pipeline import pipeline
 from .diagnose_me_cli import diagnose_me
 from .experiment import experiment
-from .output import OutputFormat
```

Comment:
@numerology / @Bobgy The line deletions on this file might need to be reverted back for the cli output format as it might cause conflicts with the linked implementations in the entire cli package.

https://github.com/kubeflow/pipelines/pull/4566